google_guest_agent/oslogin.go

```diff
@@ -188,6 +199,44 @@ func (o *osloginMgr) Set(ctx context.Context) error {
 	return nil
 }
 
+func cleanupDeprecatedLines(fpath string, directives []string) error {
+	// If the file doesn't exist don't even try updating it.
+	stat, err := os.Stat(fpath)
+	if os.IsNotExist(err) {
```

Comment:
we still need to handle other errors right?

Reply:
Sure. I'll change it, thank you.

Reply:
done. thanks.